src/plugins/text/schema.js

```diff
@@ -11,6 +11,7 @@ export default {
                 { type: 'underline' },
                 { type: 'superscript' },
                 { type: 'subscript' },
+                { type: 'code' },
```

Comment:
I think `code` should be an inline. Marks can overlap, so you could end up with e.g. an `emphasis` inside a `code` span. Inlines can (if I'm not mistaken) specify what marks they allow inside them.

Reply:
Gutenberg handle this as a mark. Slate also provide example with `code` as a mark.

Reply:
Gutenberg is Slate based? And as for Slate's example, see what happens when you add other marks inside that inline code snippet. I don't think such behaviour is acceptable.

Reply:
In both of this editor marks like `strong`, `emphasis` are allowed inside `code` mark.

Reply:
Yes, and it looks terrible and makes no sense.

Reply:
It makes sens because it's text editor and not code editor so you should always have some kind of way to make some fragment bold or italic, etc. 
It's also allowed according to the documentation.
However if you want me to make it inline then we have to first get `wrapInline()` function to work again.